tools/test_modules/m25400.pm

```diff
@@ -228,14 +237,11 @@ sub module_generate_hash
   }
 
   #printf("\$u = %s\n", unpack ("H*", $u));
-
-  $u .= substr (pack ("H*", $u_save), 16, 16);
-
   #printf("\$o = %s\n", unpack ("H*", $o));
   #printf("\$u = %s\n", unpack ("H*", $u));
 
   my $hash = sprintf ('$pdf$%d*%d*128*%d*%d*16*%s*32*%s*32*%s', $V, $R, $P, $enc, $id, unpack ("H*", $u), unpack ("H*", $o));
-
+  #print("hash\n".$hash."\n");
```

Comment:
don't only comment it out, but please completely remove it

Reply:
Agree, removed